content/en/docs/adopters.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Notation Adopters"
+description: "Organizations below all are using Notation in production."
+type: docs
+weight: 9
+---
+
+The following organizations are known adopters who use Notation in production or integrate Notation into their products and services.
+
+We are happy and proud to have you all as part of the community! To add your use case and organization to this list, please raise a pull request.
+
+## Notation Adopters
+
+All organizations are sorted alphabetically below.
+
+| Organization | Website | Scenario | 
+| --- | --- | --- | 
+| Amazon | https://aws.amazon.com/ | AWS container signing solution |
```

Comment:
@vaninrao10 Could you pls help to confirm and provide the scenario in Amazon?

Reply:
@iamsamirzon @priteshbandi Could you pls help to confirm and provide the scenario at Amazon? I added AWS Signer to this list but you can change it as you want.